docs/v3-migration-doc.md

```diff
@@ -12,112 +11,62 @@
 - [Known issues](#known-issues)
   - [Cannot delete pod if using Tailing Sidecar Operator](#cannot-delete-pod-if-using-tailing-sidecar-operator)
   - [OpenTelemetry Collector doesn't read logs from the beginning of files](#opentelemetry-collector-doesnt-read-logs-from-the-beginning-of-files)
+- [Full list of changes](#full-list-of-changes)
 
 Based on the feedback from our users, we will be introducing several changes
 to the Sumo Logic Kubernetes Collection solution.
 
 In this document we detail the changes as well as the exact steps for migration.
 
-## Changes
+## Important changes
 
-- Upgrading kube-prometheus stack
+### OpenTelemetry Collector
 
-  We are updating Kube-prometheus-stack to newest available version.
-  Major feature related to that change is upgrading kube-state-metrics to v2
+The new version replaces both Fluentd and Fluent Bit with the OpenTelemetry Collector. In the majority of cases, this doesn't
+require any manual intervention. However, custom processing in Fluentd or Fluent Bit will need to be ported to the OpenTelemetry Collector
+configuration format. See below for details.
```

Comment:
ca nwe link it to the section? `See below for details.`

Reply:
I will once that section actually exists. I'm going to add it in a new PR.